### ISSUE TITLE
Accessibility Testing for BMT Team II, Your VA Letters and Documents, Edit Mailing Address page

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#103879]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@1Copenut 

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [ ] ~~Any charts, maps, infographics, and tables convey all information without only relying on color~~
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

Color testing went well. There were no issues observed with manual or automated (axe) review of color contrast.

### Who completed the axe scans?

@1Copenut 

### axe checks

- [x] Each page has been scanned using axe (results shared below)
- [ ] ~~axe is integrated into your end-to-end testing~~

### axe DevTools scan results

The axe devtools scan came back with zero issues. The test was run with both expandable sections opened (more information about General Service address and non-US military base address).

### How did axe testing go?

The team doesn't have Cypress tests for this page yet. It's behind a feature flag so we will add them before we get to the Staging Review. These tests will evaluate the page using the same parameters as the axe devtools scan.

### Who completed the content zoom and reflow test?

@1Copenut 

### Content zoom and reflow checks

- [x] All page elements are readable and usable at 200% zoom
- [x] All page elements are readable and usable at 300% zoom
- [x] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

Zoom testing went well. The `Country` dropdown menu extended past the right edge of the browser window when opened at 400%. This is almost certainly due to the length some country names and only presents at 400% zoom.

### Who completed the keyboard navigation test?

@1Copenut 

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

Keyboard testing went well. No issues were found.

### Do you have any other results to share?

Tested with NVDA (Chrome and Firefox) and MS Voice Access. Controls were accessible and usable for both input device types (voice and screen reader + keyboard). There were two issues the BMT II team should address:

* The Edit mailing address heading does not take keyboard focus when loaded. Screen readers do not announce the route change.
* Two links say they'll open in new tabs but don't have `target="_blank"` attributes and open in the same tab.

**Comment by 1Copenut** [comment]
@salihaghaffar & @dysbo. This accessibility review of the Edit mailing address PR is complete. I created 3 issues based on findings and related them to this issue. I'm thinking we can use these individual accessibility audits (and their related findings) to show work we've done when we get to the Staging Review.
